packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -50,64 +48,61 @@ export class OrgList implements vscode.Disposable {
     }
   }
 
-  public async getAuthInfoObjects() {
-    const authFilesArray = await AuthInfo.listAllAuthorizations().catch(
+  public async getOrgAuthorizations(): Promise<OrgAuthorization[] | null> {
```

Comment:
Weird that `getOrgAuthorizations()` didn't have its return type defined.  Good catch and thanks for adding.